histomicstk/segmentationschool/segmentation_school.py

```diff
@@ -273,7 +273,7 @@ def savetime(args, starttime):
 
 
     ### Params for optimizing wsi mask cleanup ###
-    parser.add_argument('--min_size', dest='min_size', default=[30,30,24000,24000,10,10] ,type=int,
+    parser.add_argument('--min_size', dest='min_size', default=[30,30,30,30,30,30] ,type=int,
```

Comment:
@sayatmimar: can you elaborate what this default to --min_size mean? Why did it need to change?

Reply:
That's a good question, in this version of the code this min size parameter was changed for some reason. It seems that's the min size of a contour,  and contour are below this value is ignored (see [this line](https://github.com/SarderLab/Multi-Compartment-Segmentation/blob/fe260c2857471fd8ecd95f15c070b85f05f59f8d/histomicstk/segmentationschool/Codes/IterativePredict_1X.py#L469)). I did not write the code so I don't have a good answer. Maybe @njlucare knows? Or we can ask Brandon.

Reply:
Hmm I understand the line you pointed still I don't understand why the number of list elements to be 6 also what prompted changing the value from 24000 to 30, seems odd. @njlucare can you help?